client/apps/game/src/dojo/sync.ts

```diff
@@ -221,34 +199,34 @@ const configClauses: Clause[] = [
   {
     Keys: {
       keys: [WORLD_CONFIG_ID.toString()],
-      pattern_matching: "FixedLen",
+      pattern_matching: "VariableLen",
       models: [],
     },
   },
   {
     Keys: {
-      keys: [WORLD_CONFIG_ID.toString(), undefined],
+      keys: [BUILDING_CATEGORY_POPULATION_CONFIG_ID.toString(), undefined],
       pattern_matching: "FixedLen",
       models: [],
     },
   },
   {
     Keys: {
-      keys: [WORLD_CONFIG_ID.toString(), undefined, undefined],
-      pattern_matching: "FixedLen",
+      keys: [HYPERSTRUCTURE_CONFIG_ID.toString()],
+      pattern_matching: "VariableLen",
       models: [],
     },
   },
   {
     Keys: {
-      keys: [BUILDING_CATEGORY_POPULATION_CONFIG_ID.toString(), undefined],
+      keys: [undefined],
       pattern_matching: "FixedLen",
       models: [],
     },
   },
   {
     Keys: {
-      keys: [HYPERSTRUCTURE_CONFIG_ID.toString()],
+      keys: [undefined, undefined],
       pattern_matching: "VariableLen",
       models: [],
     },
```

Comment:
does this mean we sync everything in the game with this ? 

 Keys: {
      keys: [undefined],
      pattern_matching: "FixedLen",
      models: [],
    },

means everything that has 1 key

 Keys: {
      keys: [undefined, undefined],
      pattern_matching: "VariableLen",
      models: [],
    },

means everything with at least 2 keys 


is that right ?